cis/libs/encryption.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+from cis.settings import get_config
+
+class Operation(object):
+    def __init__(self, boto_session):
+        self.config = get_config()
+        self.data_key = None
+        self.iv = None
+
+        try:
+            self.kms = boto_session.client(service_name='kms')
+        except Exception as e:
+            self.kms = None
```

Comment:
This is a fall through to avoid having to patch / mock KMS.  I've left the code as is but improved the exception handler.